h2/src/main/org/h2/value/DataType.java

```diff
@@ -315,7 +315,8 @@
                 // 24 for ValueTimestamp, 32 for java.sql.Timestamp
                 56
         );
-        add(Value.TIMESTAMP_TZ, Types.OTHER, "TimestampTimeZone",
+        // TIMESTAMP_WITH_TIMEZONE = 2014
+        add(Value.TIMESTAMP_TZ, 2014, "TimestampTimeZone",
```

Comment:
please a comment here why we are doing this (I don't like to look in git history to figure out things like that)